### PR TITLE
[bug fix] some stuff

### PR DESCRIPTION
**Pull request by Faf4a**
## Type
- [x] Bug Fix

### CODE REVIEW

File: src/functions/interaction/clear.js [hunk 1]
```diff
@@ -1,56 +1,78 @@
...
+
+  messages = [...messages.values()]
+    .filter((x) => {
+      if (filters.includes("everyone")) return true;
+      if (filters.includes("notpinned") && x.pinned) return false;
+      if (filters.includes("bots") && x.author?.bot) return true;
+      if (
+        filters.some(
+          (filter) =>
+            filter.startsWith("user:") && x.author?.id === filter.split(":")[1]
```
Comment (Faf4a) [inline src/functions/interaction/clear.js:47]: Unsure about this one, if it's needed or not. However, using this slows the response down around 2-4 seconds depending on amount.

File: src/functions/interaction/clear.js [hunk 2]
```diff
@@ -1,56 +1,78 @@
...
-            "custom",
-            {},
-            "Failed To Delete Message With Reason: " + err,
-        );
+  const data = d.util.aoiFunc(d);
+  const { code } = d.command;
+  if (data.err) return d.error(data.err);
+
+  let [
+    channelID = d.channel.id,
```
Comment (Leref) [inline src/functions/interaction/clear.js]: fix format of extra space.

File: src/functions/interaction/clear.js (hunk 1 above)
Comment (Leref) [inline src/functions/interaction/clear.js:32]: `:` ?

Comment (Faf4a) [reply src/functions/interaction/clear.js:47]: user:userid,user:userid

File: src/functions/interaction/clear.js (hunk 2 above)
Comment (Faf4a) [reply src/functions/interaction/clear.js]: done

File: src/utils/CustomBox.js [hunk 3]
```diff
@@ -1,33 +1,43 @@
 const chalk = require('chalk');
```
Comment (Leref) [inline src/utils/CustomBox.js:1]: This probably should be moved entirely in util helpers or in another class of Utils so able to be exported.

**Comment by LordexDuck3990** [comment]
hi

**Pull request merged**